src/api/app/models/workflow.rb

```diff
@@ -132,4 +136,12 @@ def restore_target_projects
       processable_steps.first.create_or_update_subscriptions(target_package, filters)
     end
   end
+
+  def track_instantiation
```

Comment:
Wouldn't that be

![Screenshot from 2022-02-17 18-06-54](https://user-images.githubusercontent.com/514785/154533454-764474a8-fe97-4107-bf2b-df0cf8068425.png)


https://obs-measure.opensuse.org/d/GRaKLnm7k/scm-ci-integration?orgId=1&viewPanel=2